src/main/java/org/ladocuploader/app/preparers/ExpeditedSnapPreparer.java

```diff
@@ -0,0 +1,74 @@
+package org.ladocuploader.app.preparers;
+
+import static java.util.Collections.emptyList;
+
+import formflow.library.data.Submission;
+import formflow.library.pdf.PdfMap;
+import formflow.library.pdf.SingleField;
+import formflow.library.pdf.SubmissionField;
+import formflow.library.pdf.SubmissionFieldPreparer;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+import software.amazon.awssdk.services.kms.endpoints.internal.Value.Int;
```

Comment:
Not really reviewing, just passing through.  This is probably not the thing to use here. I'd point you more towards Integer.parseInt() / Integer.valueOf().

Reply:
Although maybe this is an unused import all together.